constant/styles/fonts.ts

```diff
@@ -5,8 +5,8 @@ const bold = 700;
 
 const family = "Halyard Display";
 
-export const headerXl = `${bold} 80px/110px ${family}`;
-export const headerLg = `${bold} 40px/55px ${family}`;
+export const headerXl = `${bold} clamp(40px, 10vw, 80px)/clamp(55px, 8vw, 110px) ${family}`;
```

Comment:
This is fluid typography. We clamp the size of the headings so that they don't get too big or too small. They react to the screen size appropriately without needing any media queries.